app/scripts/modules/core/src/pipeline/manualExecution/ManualPipelineExecutionModal.tsx

```diff
@@ -340,74 +348,76 @@ export class ManualExecutionModal extends React.Component<IManualExecutionModalP
               <Modal.Title>{modalHeader}</Modal.Title>
             </Modal.Header>
             <Modal.Body>
-              <div className="container-fluid modal-body-content">
-                {pipelineOptions.length > 0 && (
-                  <PipelineOptions
-                    formik={formik}
-                    formatPipeline={this.formatPipeline}
-                    formatTriggers={this.formatTriggers}
-                    formatParameterConfig={this.formatParameterConfig}
-                    pipelineOptions={pipelineOptions}
-                    pipelineChanged={this.pipelineChanged}
-                    triggerChanged={this.triggerChanged}
-                    updateTriggerOptions={this.updateTriggerOptions}
-                  />
-                )}
-                {formik.values.pipeline && (
-                  <div className="form-group">
-                    <div className="col-md-10">
-                      <p>
-                        This will start a new run of <strong>{formik.values.pipeline.name}</strong>.
-                      </p>
+              <LayoutProvider value={ManualExecutionFieldLayout}>
+                <div className="container-fluid modal-body-content">
+                  {pipelineOptions.length > 0 && (
+                    <PipelineOptions
+                      formik={formik}
+                      formatPipeline={this.formatPipeline}
+                      formatTriggers={this.formatTriggers}
+                      formatParameterConfig={this.formatParameterConfig}
+                      pipelineOptions={pipelineOptions}
+                      pipelineChanged={this.pipelineChanged}
+                      triggerChanged={this.triggerChanged}
+                      updateTriggerOptions={this.updateTriggerOptions}
+                    />
+                  )}
+                  {formik.values.pipeline && (
+                    <div className="form-group">
+                      <div className="col-md-6 col-md-offset-4">
```

Comment:
nothing new, but this has looked really weird forever and I wanted to change it. It didn't respect the overall layout boundaries of the rest of the modal and always felt off.